src/psycopmlutils/loaders/synth/raw/load_synth_data.py

```diff
@@ -16,7 +16,7 @@ def load_raw_test_csv(filename: str, n_rows: Optional[int] = None) -> pd.DataFra
         n_rows (int, optional): Number of rows to load. Defaults to None.
     """
     # Get project root dir
-    project_root = Path(__file__).resolve().parents[5]
+    project_root = Path()
```

Comment:
I think this might cause an error if not called from the project root dir

Reply:
my comment on Lasses suggestion:

>  It [Path()] gives you ".", which is relative to where you are running it from, not from the file.

Reply:
Yeah, seems like the most robust solution would be to use `__file__`. To avoid potential issues if moving files around, we specifly a `PROJECT_ROOT` in a main `utils.py` file and import that throughout

Reply:
PROJECT_ROOT seems like the way to go. I'll add it.